src/main/java/org/jabref/gui/shared/SharedDatabaseLoginDialogView.java

```diff
@@ -0,0 +1,320 @@
+
+    private final SharedDatabasePreferences prefs = new SharedDatabasePreferences();
+
+    private final ButtonType connect = new ButtonType("Connect", ButtonData.YES);
+    private final Button btnConnect;
+    private final JabRefFrame frame;
+
+    private DBMSConnectionProperties connectionProperties;
+    private SharedDatabaseLoginDialogViewModel viewModel;
+
+    public SharedDatabaseLoginDialogView(JabRefFrame frame) {
+        this.frame = frame;
+        this.setTitle(Localization.lang("Connect to shared database"));
+
+        this.getDialogPane().getButtonTypes().addAll(connect, ButtonType.CANCEL);
+        this.btnConnect = (Button) this.getDialogPane().lookupButton(connect);
+        btnConnect.setOnAction(this::openDatabase);
+
+        ViewLoader.view(this)
+                  .load()
+                  .setAsContent(this.getDialogPane());
+
+    }
+
+    private void openDatabase(ActionEvent e) {
+
+        try {
+            checkFields();
+
+            connectionProperties = new DBMSConnectionProperties();
+            connectionProperties.setType(cmbDbType.getSelectionModel().getSelectedItem());
+            connectionProperties.setHost(tbHost.getText());
+            connectionProperties.setPort(Integer.parseInt(tbPort.getText()));
+            connectionProperties.setDatabase(tbDb.getText());
+            connectionProperties.setUser(tbUser.getText());
+            connectionProperties.setPassword(tbPwd.getText());
+
+            openSharedDatabase();
+        } catch (JabRefException exception) {
+            dialogService.showErrorDialogAndWait(Localization.lang("Warning"), exception);
+
+        }
+    }
+
+    @FXML
+    private void initialize() {
+
+        viewModel = new SharedDatabaseLoginDialogViewModel();
+        cmbDbType.itemsProperty().bind(viewModel.dbmstypeProperty());
+        cmbDbType.getSelectionModel().select(0);
+        viewModel.selectedDbmstypeProperty().bind(cmbDbType.getSelectionModel().selectedItemProperty());
+
+        viewModel.selectedDbmstypeProperty().addListener((observable, oldValue, newValue) -> {
+            tbPort.setText(Integer.toString(newValue.getDefaultPort()));
+        });
+
+        btnBrowse.disableProperty().bind(chkAutosave.selectedProperty().not());
+        tbFolder.disableProperty().bind(chkAutosave.selectedProperty().not());
+
+        applyPreferences();
+    }
+
+    @FXML
+    void openFileDialog(ActionEvent event) {
+        FileDialogConfiguration fileDialogConfiguration = new FileDialogConfiguration.Builder()
+                                                                                               .addExtensionFilter(FileType.BIBTEX_DB)
+                                                                                               .withDefaultExtension(FileType.BIBTEX_DB)
+                                                                                               .withInitialDirectory(Globals.prefs.get(JabRefPreferences.WORKING_DIRECTORY))
```

Comment:
Please extract the `Globals.prefs` dependency. The new JavaFX stuff should no longer depend on `Globals`.